### PR TITLE
Add mobile module

### PR DESCRIPTION
**Pull request by damdafayton**
`index` and `about` html pages addes.
Need to click on `Our Team` to reach About Us page for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +39,6 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 
-## Acknowledgments
```
Comment (t-yanick) [inline README.md:47]: - [ ] As it is a requirement for this project and best practice, please kindly acknowledge the author of your design.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>About Us & Our Team</title>
+</head>
+
+<body>
+    <main class="container-fluid bg-gray row mx-0">
```
Comment (t-yanick) [inline about.html:14]: - [ ]  Please kindly add   a menu with links to the various pages, i.e `Home` and `About` page. Please make sure this pages and present and provide links to them in the menu bar as in the design. Please kindly fix that.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <section class=" container gap-4" id="team">
+            <a class="h2-wrapper" href="about.html">
+                <h2>Our Team</h2>
+                <div class="line"></div>
+            </a>
+        </section>
+    </main>
+    <footer class="copyright py-4 text-start d-flex justify-content-center">
+        <p class="col-8">2021,<br>1065 Ave of the Americas, New York, NY 10018,<br>BlaBla Restaurant</p>
+    </footer>
```
Comment (t-yanick) [inline about.html:41]: - [ ] The navigation bar and the footer is not displaying on your pages, please kindly fix the issue.

File: about.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>About Us & Our Team</title>
+</head>
+
```
Comment (t-yanick) [inline about.html:12]: - [ ] The mobile version of your app is not implemented, Please add that.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @damdafayton , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

### Please  your project is missing quite a handful of requirements, I will just summerize them here ###
- [ ] Please kindly add a video presenting the features of your project, and add a link to the video in your `readme file`.
- [ ] Please kindly a navigation bar, with links to `Home` and `About` pages
- [ ] Make sure your site has two pages, `Home` and `About` page and should contain the same navbar and footer that matches the design. 
- [ ] Kindly implement the desktop size as per the design.
- [ ] Kindly implement the mobile size as per the design.
- [ ] The footer should contain a logo aligned to the left, and text aligned to the right. 
- [ ] Please make sure the design and styling matches the template. 
- [ ] Please make sure your CSS styling is built as mobile-first, media queries are used to change the design from mobile to desktop, not the other way around.
- [ ] Please make sure media queries are properly organized at the bottom of the stylesheet.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 3 above)
Comment (damdafayton) [reply about.html:41]: Hi, this and almost all your other comments are not related to today's task. I think you make evaluation based on wrong criteria.

**Pull request merged**